chain-vote/src/lib.rs

```diff
@@ -219,11 +219,11 @@ fn group_elements_from_bytes(bytes: &[u8]) -> Option<Vec<gang::GroupElement>> {
     Some(elements)
 }
 
-pub fn result(
```

Comment:
Please also rename the usage in `chain-impl-mockchain`.

Reply:
Is it needed though if we plan to replace that with the validation function on the chain? I was waiting for that change before merging this